__tests__/app.test.js

```diff
@@ -22,19 +22,71 @@ describe('GET/api/categories', () => {
             let resultArray = response.body
 
             expect(resultArray.length).toBe(4) 
-
             resultArray.forEach((category) => {
                 expect(category).toHaveProperty("slug")
                 expect(category).toHaveProperty("description")
             })
         })
     });
- 
 });
 
-describe('Deals with api paths that does not exist', () => {
+describe('Error Handler - Deals with api paths that does not exist', () => {
     test('resolves with 404: Not found, for routes that does not exist', () => {
-        return request(app).get('/invalidAPI').expect(404)
+        return request(app).get('/invalidAPI').expect(404).then((response) => {
+            expect(response.body).toEqual({msg: 'Invalid API path'})
+        })
+    })
+});
+
+describe('GET/api/reviews', () => {
+    test('should return a status code of 200', () => {
+        return request(app).get('/api/reviews').expect(200)
+    });
+    test('should return an array', () => {
+        return request(app).get('/api/reviews').then(({body}) => {
+            expect(Array.isArray(body)).toBe(true)
+        })
+    });
+    test('each element of the array should contain following properties', () => {
+        return request(app).get('/api/reviews').then(({body}) => {
+            expect(body.length).toBe(13)
+            body.forEach((review) => {
+                expect(review).toHaveProperty("owner")
+                expect(review).toHaveProperty("title")
+                expect(review).toHaveProperty("review_id")
+                expect(review).toHaveProperty("category")
+                expect(review).toHaveProperty("review_img_url")
+                expect(review).toHaveProperty("created_at")
+                expect(review).toHaveProperty("votes")
+                expect(review).toHaveProperty("designer")
+                expect(review).toHaveProperty("comment_count")
+            })
         })
     });
+    test('comment_count for review_id 2,3 = 3', () => {
```

Comment:
amending the test above would be enough to alleviate the need for both of these tests imo